tap_github/repository_streams.py

```diff
@@ -1907,6 +1907,457 @@ def query(self) -> str:
+                  node_id: id
+                  id: databaseId
+                  number
+                  title
+                  body: bodyText
+                  url
+                  created_at: createdAt
+                  published_at: publishedAt
+                  last_edited_at: lastEditedAt
+                  answer_chosen_at: answerChosenAt
+                  updated_at: updatedAt
+                  closed_at: closedAt
+
+                  created_via_email: createdViaEmail
+
+                  author {
+                  ... on User{
+                    node_id: id
+                    id: databaseId
+                    login
+                    avatar_url: avatarUrl
+                    html_url: url
+                    type: __typename
+                    site_admin: isSiteAdmin
+                    }
+                  }
+                  author_association: authorAssociation
+                  category {
+                    node_id: id
+                    slug
+                    name
+                    description
+                  }
+                  labels(first: 100) {
+                    edges {
+                      node {
+                        node_id: id
+                        created_at: createdAt
+                        name
+                        description
+                        url
+                        resource_path: resourcePath
+                        color
+                        default: isDefault
+                      }
+                    }
+                  }
+                  locked
+                  active_lock_reason: activeLockReason
+                  closed
+                  is_answered: isAnswered
+                  answer {
+                    id: databaseId
+                    node_id: id
+                    body
+                    author {
+                    ... on User{
+                      node_id: id
+                      id: databaseId
+                      login
+                      avatar_url: avatarUrl
+                      html_url: url
+                      type: __typename
+                      site_admin: isSiteAdmin
+                      }
+                    }
+                    author_association: authorAssociation
+                  }
+                  answer_chosen_by: answerChosenBy {
+                  ... on User{
+                    node_id: id
+                    id: databaseId
+                    login
+                    avatar_url: avatarUrl
+                    html_url: url
+                    type: __typename
+                    site_admin: isSiteAdmin
+                    }
+                  }
+                  upvote_count: upvoteCount
+                  reactions(first: 100) {
+                    edges {
+                      node {
+                        reaction_type: content
+                        reacted_at: createdAt
+                        user {
+                          login
```

Comment:
I wonder if we should grab the same user fields we grab everywhere else, for tap consistency. 🤔